models/basgra/inst/template.job

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# redirect output
+exec 3>&1
+exec &> "@OUTDIR@/logfile.txt"
+
+# host specific setup
+@HOST_SETUP@
+
+# create output folder
+mkdir -p "@OUTDIR@"
+
+# see if application needs running
+if [ ! -e "@OUTDIR@/results.csv" ]; then
+    
+  "@BINARY@"
```

Comment:
I think you call the model below, so this can be removed? Or am I missing something?

```suggestion
```